src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
-export interface Figure {
+type ColorType = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: string,
+  color: ColorType,
+  getArea(): number;
 }
 
 export class Triangle {
+  shape = 'triangle';
```

Comment:
enum is much better when you need to hardcode values, because in the case when you need to change the value, you need to do it in only one place